.ci/lib.sh

```diff
@@ -246,6 +246,31 @@ delete_stale_kata_resource()
 	done
 }
 
+delete_kata_repo_registrations()
+{
+	case "$ID" in
+		ubuntu | debian)
+			local apt_file="/etc/apt/sources.list.d/kata-containers.list"
+			if [ -f "$apt_file" ]; then
+				info "Removing Kata apt file [$apt_file]"
+				sudo rm -f "$apt_file"
+			fi
+			# apt-key output format changed at ubuntu 16.10
+			if [ "$VERSION_ID" < "16.10" ]; then
```

Comment:
Alas this won't work as test only handles integers. The warning we get from `apt-key list` is clearly suggesting we don't do what we're trying to here, presumably because the format may change (as it has). But if that's the case, what *is* the correct way of parsing such info?

Reply:
Ah - I see the problem - this fails currently as the shell treats `<` as a redirection (try it out ;)

The fix - quote the operator:

```sh
[ "$VERSION_ID" \< "16.10" ]
```

Reply:
huh! wah wah, oh - let me look into that - bizzaro! thx!

Reply:
heh, yep, need the escape with `[` but not `[[`... apparently, maybe.
escape added, debian dropped.
29th attempt gotta be the charm - let's try again...